conf/modules/snv_annotation.config

```diff
@@ -39,10 +39,10 @@ process {
     withName: '.*:SNV_ANNOTATION:ENSEMBLVEP_VEP' {
         ext.prefix = { "${meta.id}_vep" }
         ext.args = { [
-            "--dir_plugins ${cache}/Plugins",
-            "--plugin LoFtool,${cache}/LoFtool_scores.txt",
-            "--plugin pLI,${cache}/pLI_values.txt",
-            "--plugin SpliceAI,snv=${cache}/spliceai_21_scores_raw_snv_-v1.3-.vcf.gz,indel=${cache}/spliceai_21_scores_raw_snv_-v1.3-.vcf.gz",
+            "--dir_plugins .",
+            "--plugin LoFtool,LoFtool_scores.txt",
+            "--plugin pLI,pLI_values.txt",
+            "--plugin SpliceAI,snv=spliceai_21_scores_raw_snv_-v1.3-.vcf.gz,indel=spliceai_21_scores_raw_snv_-v1.3-.vcf.gz",
```

Comment:
I guess that this is only the scores for chr21? In that case would it be worth having the presumptive path to the full file here and in the test config use the test file so that the user don't accidentally use the test file in a full run? Or maybe this is covered in the documentation already.

Reply:
Good point, I didn't realise that these are just 21 and I have been using the test file in a full run 😄 I will remove this SpliceAI from the config and documentation.

Reply:
Same for raredisease then? 

https://github.com/nf-core/raredisease/blob/c7a6f389c9b958bf5099c1f48c9b8145d755510f/conf/modules/annotate_genome_snvs.config#L90

Reply:
I would assume so